templates/base.html

```diff
@@ -6,286 +6,170 @@
+      <thead>
+        <tr>
+          <td />
+          <td style="height: 130px">
+            <table style="margin-left: auto; margin-right: auto">
+              <tbody>
+                <tr>
+                  <td>
+                    <a href="https://about.codecov.io/">
+                      <img
+                        src="https://app.codecov.io/logo-192x192.png"
+                        alt="Codecov logo"
+                        width="80px"
+                        height="80px"
+                      />
+                    </a>
+                  </td>
+                  <td style="padding-left: 20px">
+                    <h1 style="font-size: 48px; font-weight: 700; margin: 0">
+                      Codecov
+                    </h1>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </td>
+          <td />
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td />
+          <td style="height: 130px; width: 600px">
+            {% block main %}{% endblock %}
+          </td>
+          <td />
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <td colspan="3">
+            <table
+              style="
+                width: 100%;
+                border-top: 1px solid rgb(216, 220, 226);
+                padding-top: 16px;
+              "
+            >
+              <tbody>
+                <tr>
+                  <td colspan="5" style="text-align: center">
+                    <h1
+                      style="
+                        margin: 0;
+                        line-height: 45px;
+                        font-size: 30px;
+                        font-weight: 700;
+                        color: rgb(57, 71, 84);
+                      "
+                    >
+                      Codecov
+                    </h1>
+                    <p
+                      style="
+                        font-weight: 400;
+                        font-size: 18px;
+                        line-height: 27px;
+                        margin: 0;
+                        color: rgba(57, 71, 84);
+                      "
+                    >
+                      Functional Software, dba Sentry
+                    </p>
+                    <p
+                      style="
+                        font-weight: 400;
+                        font-size: 18px;
+                        line-height: 27px;
+                        margin: 0;
+                        color: rgba(153, 159, 167);
+                      "
+                    >
+                      45 Fremont St. 8th Floor, San Francisco, CA 94105
+                    </p>
+                  </td>
+                </tr>
+                <tr style="height: 25px" />
+                <tr style="text-align: center">
+                  <!-- these svgs don't work, to get this looking right we'd need to get PNGs served by CDN, perhaps we add these links to our app footer? -->
```

Comment:
Coming back to these in the future

Reply:
fine to send em for now, SVGs do work on a few email clients (just not popular ones)